### DISCUSSION TITLE
macOS: weird background color when opening fullscreen

### DISCUSSION THREAD

**Discussion by tindt4869** [ghostty-org/ghostty#3991] [Q&A]
When entering fullscreen mode, the ghostty background color look like my LED screen died.

### Screenshots:

**Before fullscreen**
<img width="1440" alt="Screenshot 2024-12-30 at 10 33 30" src="https://github.com/user-attachments/assets/c2de44d4-ea20-4d24-a592-8978fb352cb7" />

**After fullscreen**
<img width="1440" alt="Screenshot 2024-12-30 at 10 33 21" src="https://github.com/user-attachments/assets/f9ba521d-618a-40c5-a5e7-6881c4b0600b" />

**My ghostty config**
```conf
theme = rose-pine-dawn
adjust-cell-height = 20%
font-size = 13.5
```

**Comment by mitchellh** [comment]
Are you on an Intel Mac? There's a known Metal GPU driver bug on Intel that causes this.

**Comment by sitsofe** [comment]
(This sounds similar to #3352)
